remo/profiles/templates/profiles_mentors.jinja

```diff
@@ -0,0 +1,51 @@
+{% extends "base_fd4.jinja" %}
+{% block pagetitle %}Mozilla Reps - Mentors{% endblock %}
+
+{% block logo %}
+<h1><span class="metabold">mozilla</span> reps</h1>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="mentors-main">
+  <div class="row">
+    <div class="large-12 columns">
+      <h1>Mentors</h1>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="large-12 columns">
+      {% for mentor in mentors %}
+        <h3>{{mentor}} ({{mentor.mentees.count()}})</h3>
+        <table class="dashboard-table responsive">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Country</th>
+              <th>Date joined</th>
+              <th>Reports past 3 months</th>
+              <th>Reports past 6 months</th>
+              <th>Reports past 12 months</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for rep in mentor.mentees.exclude(user__groups__name='Alumni') %}
```

Comment:
This also displays users who have the mentor bit in their profile but have not mentees. Probably this is an edge case and I am not sure if there are cases like that in prod. Do we want this?

Reply:
Yes, that is something we want to know about. However, since there is no table row in that case, the styling looks sub-optimal. I'll fix this.